main.rb

```diff
@@ -1,12 +1,27 @@
-# main.rb
-
-require_relative 'person'
-require_relative 'capitalize_decorator'
-require_relative 'trimmer_decorator'
-
-person = Person.new(22, 'maximilianus')
-person.correct_name
-capitalized_person = CapitalizeDecorator.new(person)
-puts capitalized_person.correct_name
-capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
-puts capitalized_trimmed_person.correct_name
+#!/usr/bin/env ruby
+
+require_relative 'app'
+require_relative 'menu'
+require_relative 'actions'
+
+def main
+  welcome_message
+
+  loop do
+    display_options
+
+    option = gets.chomp.to_i
+
+    case option
+    when 1 then create_student
+    when 2 then create_teacher
```

Comment:
- [ Required ] I think it would be nice if these two options were collapsed into one single choice `Create a person` , when a user chooses the `Create a person` option, then the application should give the user a choice of creating either a `Student` or a `Teacher`. This is how this functionality has been implemented in the in _project video_ provided for this milestone. Please see the screenshot below and rectify this.

## Project requirements
![Screenshot from 2023-11-10 18-11-24](https://github.com/martinkarugaba/oop-school-library/assets/16392046/7f14c8a8-7d0b-4068-b994-1236481f4618)

## Your design
![Screenshot from 2023-11-10 18-13-38](https://github.com/martinkarugaba/oop-school-library/assets/16392046/2b3add2c-da4d-45a6-a674-480c475d99fb)